### ISSUE TITLE
Discussion: front-end js only

### ISSUE THREAD

**Issue by sirkitree** [davexoxide/rendervideositemap#4]
I can't help but wonder if the node server is actually needed here. Could this be built with just react?

**Comment by davexoxide** [comment]
Yes, yes it can! Totally open to exploring this for more refined application.

**Comment by sirkitree** [comment]
i have a feeling that a front-end only solution may not be able to pull the xml cross site, so the back-end may be needed to first download the xml files for the front-end to parse.

Still would be good to try
